src/variants.ini

```diff
@@ -335,12 +302,7 @@ promotionPieceTypes = q
 doubleStep = false
 castling = false
 
-# Centaurking
-# A variant demonstrating how to define a custom royal piece movement
-[centaurking:chess]
```

Comment:
Please add this one back, because it was added explicitly to show the custom king feature.